packages/frontend/web-ui/src/game/hooks/useGame.spec.ts

```diff
@@ -79,6 +85,10 @@ describe(useGame.name, () => {
         switchCardSelection: jest.fn(),
       };
 
+      useGetFinishedGameWinnerResultFixture = {
+        endGameWinner: undefined,
```

Comment:
```suggestion
        finishedGameWinner: undefined,
```

`finished` is the word we are using. An endgame is widely used to refer to the final stage of a game.